app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,68 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions.length">
+  <button
+    uib-dropdown-toggle
+    class="btn btn-link dropdown-toggle"
+    type="button"
+    id="dropdownKebabRight-{{$id}}"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="true">
+    <span class="fa fa-ellipsis-v"></span>
+  </button>
+  <ul
+    class="dropdown-menu dropdown-menu-right"
+    aria-labelledby="dropdownKebabRight">
+    <li
+      ng-repeat="action in notification.actions"
+      role="{{action.isSeparator === true ? 'separator' : 'menuitem'}}"
+      ng-class="{'divider': action.isSeparator === true, 'disabled': action.isDisabled === true}">
+      <a
+        ng-if="!action.isSeparator"
+        class="secondary-action"
+        title="{{action.title}}"
+        ng-click="$ctrl.customScope.handleAction(notification, action)">
+        {{action.name}}
+      </a>
+    </li>
+  </ul>
+</div>
+
+
+<span
+  ng-if="notification.event"
+  class="{{'pull-left ' + $ctrl.customScope.getNotficationStatusIconClass(notification.event)}}"
+  ng-click="$ctrl.customScope.markRead(notification)"></span>
```

Comment:
Can do.  Ends up with 3 items reachable by tab:
- row to mark read
- name of object to view it 
- date/time to clear.  This doesn't necessarily make sense, there is `sr-only` text, but to a user who is not using a reader, its odd.

![screen shot 2017-08-14 at 1 34 17 pm](https://user-images.githubusercontent.com/280512/29283636-9512e2b6-80f5-11e7-8b3d-a640d3222e94.png)
![screen shot 2017-08-14 at 1 34 22 pm](https://user-images.githubusercontent.com/280512/29283637-95172632-80f5-11e7-9443-0cd12891a181.png)
![screen shot 2017-08-14 at 1 36 36 pm](https://user-images.githubusercontent.com/280512/29283647-9f980d7e-80f5-11e7-82f8-99c462e00522.png)
Tinkering with ways to improve this.